index.bs

```diff
@@ -2242,6 +2253,11 @@ When this method is invoked, the user agent MUST execute the following algorithm
                         ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of
                             <code>|assertionCreationData|.[=assertionCreationData/signatureResult=]</code>.
 
+                        :   {{AuthenticatorAssertionResponse/authenticatorAttachment}}
+                        ::  If <code>|assertionCreationData|.[=assertionCreationData/authenticatorAttachmentResult=]</code> is null, set this
+                            field to null. Otherwise, set this field to a new {{DOMString}}, containing the value of
+                            <code>|assertionCreationData|.[=assertionCreationData/authenticatorAttachmentResult=]</code>.
+
```

Comment:
You need to add a similar piece of text to Create, i.e. section 5.1.3, step 20. "If any authenticator indicates success".